packages/fluentui/react-northstar-prototypes/src/prototypes/mentions/dataMocks.ts

```diff
@@ -10,13 +10,13 @@ export interface AtMentionItem {
 export const atMentionItems: AtMentionItem[] = _.times(10, () => ({
   header: `${faker.name.firstName()} ${faker.name.lastName()}`,
```

Comment:
Should we update this to use names and titles from other examples?

```ts
export const atMentionItems: AtMentionItem[] = [
  { header: 'Robin Counts', image: 'https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/RobinCounts.jpg', content: 'Software Engineer' },
  // etc.
]
```

Reply:
@layershifter  - what do you mean?